diff --git a/scripts/autogen-version b/scripts/autogen-version
index 47ab0228..e19509f4 100755
--- a/scripts/autogen-version
+++ b/scripts/autogen-version
@@ -54,16 +54,10 @@ get_version() {
         hash=$(git rev-parse --short "${ref}")
         branch=$(git symbolic-ref --short "${ref}" 2>/dev/null || echo "${hash}")
         git_version=$(git describe --always --match "v*" "${describe_arg}" | sed 's/^v//g')
-    elif [ -f ./VERSION ]; then
-        # Read information from VERSION file which contains values for `branch`, `hash`, and `git_version`.
-        . ./VERSION
-    else
-        >&2 echo "Cannot extract version information: neither git repository nor VERSION file present"
-        exit 1
     fi
 }
 
-get_version "$@"
+get_version "2020"
 
 # When running from CI, for geting the branch name we prefer what
 # might be passed in through environment variables as we may not
